template/tufte.latex

```diff
@@ -341,7 +341,7 @@ $if(it.ingredients)$
 \begin{itemize}
 $for(it.ingredients)$
   \item{%
-    $it.name$$if(it.amount)$ ($it.amount$$if(it.unit)$ $it.unit$$endif$)$endif$$if(it.processing)$---$it.processing$$endif$$if(it.notes)$\,\sidenote[][]{$it.notes$}$endif$
+  $it.name$ $if(it.amount)$($it.amount$$if(it.unit)$$it.unit$$endif$)$elseif(it.amounts)$($for(it.amounts)$$it.amount$$if(it.unit)$$it.unit$$endif$$sep$, $endfor$)$endif$$if(it.processing)$---$it.processing$$endif$$if(it.notes)$\,\sidenote[][]{$it.notes$}$endif$
```

Comment:
Looks like you ate a space with this change. There should a be whitespace between the $it.amount and the $it.unit.

This is what a recipe looks like with this PR applied:
![image](https://user-images.githubusercontent.com/6888381/109819139-a6def680-7c2b-11eb-8a32-4e224df3ff7e.png)

Reply:
Opps it looks like I missed a space I shall fix it.